Sessions/1c-Intro-to-Python.ipynb

```diff
@@ -291,66 +243,44 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With functions you can group a sequence of statements (and execute them with one function call).\n",
+    "With functions, you can **group a sequence of statements** (and execute them with one function call).\n",
     "\n",
-    "A common example in text mining is called text preprocessing, where we transform an input text to format that makes it more amenable to computational analysis. We discuss this in more detail later, but a simple preprocessing stap could be:\n",
+    "A common example in text mining is called text **preprocessing**, where we transform an input text into a format that makes it more amenable to computational analysis. We discuss this in more detail later, but a simple preprocessing step could be:\n",
     "\n",
-    "- Check if text is if minimal length 5\n",
-    "- if so, return the text in lowercased form\n",
-    "- Otherwise return and empty string"
+    "- Check if a text has more than 5 characters\n",
+    "- if so, return the text in the uppercased form (see the `.upper()` example in 1b)\n",
+    "- Otherwise, return an empty string"
```

Comment:
Hi! We're doing string functions on Tuesday, so `upper()` does not appear until notebook 2a.

Reply:
Hmm... you're right. I thought there was one example. I'll adjust the text.